### PR TITLE
Added more descriptions to README

### PR DESCRIPTION
**Pull request by alphanyang**
As requested

### CODE REVIEW

File: app/src/main/java/com/example/twitchdropsapplication/Constants.kt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package com.example.twitchdropsapplication
 
 // https://dev.twitch.tv/console
-const val TWITCH_CLIENT_ID = "YOUR_CLIENT_ID"
-const val TWITCH_CLIENT_SECRET = "YOUR_CLIENT_SECRET"
-const val TWITCH_REDIRECT_URI = "YOUR_REDIRECT_URI"
+const val TWITCH_CLIENT_ID = "vy59zs1d00ar1sz88qj61pi03ssaf0"
+const val TWITCH_CLIENT_SECRET = "ENTER SECRET"
+const val TWITCH_REDIRECT_URI = "https://twitchdropsapplication"
```
Comment (Jonas-I) [inline app/src/main/java/com/example/twitchdropsapplication/Constants.kt:6]: We probably shouldn't display our client ID or URI. Maybe we can ignore this file? @josh-improgo

File: app/src/main/java/com/example/twitchdropsapplication/composables/Composables.kt [hunk 2]
```diff
@@ -12,13 +12,14 @@ import coil.request.ImageRequest
 import com.example.twitchdropsapplication.Spacing
 
 @Composable
-fun StreamRow(channelName: String, image: String) {
+fun StreamRow(channelName: String, image: String, userProfileImage: String) {
```
Comment (Jonas-I) [inline app/src/main/java/com/example/twitchdropsapplication/composables/Composables.kt:15]: If we plan on using only the `userProfileImage`, we don't need to add another parameter here.

File: app/src/main/java/com/example/twitchdropsapplication/Constants.kt (hunk 1 above)
Comment (josh-improgo) [reply app/src/main/java/com/example/twitchdropsapplication/Constants.kt:6]: Yep, we need to keep these private. Added files to the `.gitignore` here: https://github.com/JJAlpha/TwitchClone/pull/5

We will need to create our own personal `Constants.kt` file under the `/src` directory.

File: app/src/main/java/com/example/twitchdropsapplication/MainActivity.kt [hunk 3]
```diff
@@ -47,15 +48,19 @@ class MainActivity : ComponentActivity() {
...
                         Text("User: ${authInfo.value?.login ?: "Not Signed In"}")
                         if (listOfStreamsShown.isEmpty()) {
                             Text("No Streams Available :(")
                         } else {
                             Text("Streams:")
-                            listOfStreamsShown.forEach {
-                                StreamRow(channelName = it.user_name, image = it.thumbnail_url)
+                            listOfStreamsShown.forEach { stream ->
+                                val user = listOfUsers.find { user -> user.id == stream.id }
+                                val userProfileImage = user?.profile_image_url ?: ""
```
Comment (josh-improgo) [inline app/src/main/java/com/example/twitchdropsapplication/MainActivity.kt:61]: If the `profile_image_url` returns null, let's just keep it null. No need for the empty string.

Then, when you pass `userProfileImage` for `StreamRow`, make the `userProfileImage` parameter in the method signature to be nullable.

After that, you can do a check to see if `userProfileImage` is null in the component. If it is null, then don't render the `Image` composable

File: app/src/main/java/com/example/twitchdropsapplication/composables/Composables.kt (hunk 2 above)
Comment (alphanyang) [reply app/src/main/java/com/example/twitchdropsapplication/composables/Composables.kt:15]: https://github.com/JJAlpha/TwitchClone/pull/8/commits/e88fc5c6c2d98d4239ad8278aea97102f6e73a2c

File: app/src/main/java/com/example/twitchdropsapplication/MainActivity.kt (hunk 3 above)
Comment (alphanyang) [reply app/src/main/java/com/example/twitchdropsapplication/MainActivity.kt:61]: Removed and addressed here: https://github.com/JJAlpha/TwitchClone/commit/e88fc5c6c2d98d4239ad8278aea97102f6e73a2c#diff-3baf313614318fb713b6afc46c467702b30c6e4ef55a292d899f8112b8022d28

**Comment by alphanyang** [comment]
Closed in favor of #8

**Pull request closed without merging**